scripts/cluster/common/utils.py

```diff
@@ -169,7 +169,7 @@ def is_same_server(hostname, ip):
         hname, _, _ = socket.gethostbyaddr(ip)
         if hname == hostname:
             return True
-    except:
+    except socket.error:
```

Comment:
Are we sure this will raise only a socket.error? https://docs.python.org/3/library/socket.html#socket-families

Reply:
The other exceptions that I see on that page (`socket.herror`, `socket.gaierror`, and `socket.timeout`) all subclasses from `OSError`, which `socker.error` is an alias of, so this should catch any errors that we care about.